<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<title>iEvoBio: Challenge: Synthesizing phylogenies</title>
 
 <link href="ievobio_style3.css" rel="stylesheet" type="text/css" />
<link rel="SHORTCUT ICON" href="http://www.ievobio.org/ievobio.ico"/>
<link href="print.css" rel="stylesheet" type="text/css" media="print" />

<script src="http://widgets.twimg.com/j/2/widget.js"></script>

</head>

<body>
<div id="container">
  <div id="content">
    <div id="top"><a href="index.html"><img src="images/logo2012.png" alt="iEvoBio: Informatics for Phyogenetics, Evolution, and Biodiversity Conference - Ottawa, Canada. Theme: Sythesizing Trees" width="351" height="84" border="0" /></a><a href="index.html"><img src="images/det0001_184p.jpg" alt="graft detail" width="184" height="100" border="0" /></a>
        <div class="rightinner">iEvoBio <br />
          10-11 July 2012<br />
          Ottawa, Canada<br />
          Sister meeting with <a href="http://www.evolution2012.org" target="outlink">Evolution 2012 </a></div>
    </div>
    <p>&nbsp;</p>
    <div id="main">
      <div align="right"><a href="#disqus_thread" data-disqus-identifier="challenge_comments" class="bio">Comments</a></div>
      <h2><a name="challenge" id="challenge"></a><span id="internal-source-marker_0.10428917457112141">iEvoBio 2012 Challenge: Synthesizing phylogenies</span></h2>
      <p><span id="internal-source-marker_0.10428917457112141"><em><img src="images/bonsai-tree.jpg" alt="tree" width="50" height="50" class="floatimg" />Many trees enter. </em></span><em><br />
        Fresh analysis ensues.<br />
        New insights emerge.</em><br />
        <br />
        Somewhere,   buried in large sets of trees, lies a stunning new revelation, a   baffling discovery, the answer to a longstanding controversy, or simply   something not obvious to the naked eye. The mission of the 2012 iEvoBio   challenge is to find those revelations, discoveries and answers within   your own data and/or within one of the datasets provided by the   challenge. &nbsp;What new scientifically interesting results can you pull   from these trees, using any combination of techniques at your disposal?
</p>
      <p> Here are the rules of this challenge:</p>
      
      <ol> <li>The set of trees you use must have at least 10,000 leaves in  total. Acceptable entries could be a set comprising 2,500 distinct  trees covering the same four taxa, a single tree with 10,000+ leaves,  or anything in between. </li>
      <li>Your results must be scientifically new.</li>
      <li>The data, or at least a description of the data, must be  publicly available. If working with your own dataset, you must at  least provide a summary of the data you used (see below for the  minimum description that must be provided). </li>
      <li>The source code of any tool and/or method developed as part of  your challenge submission must be publicly downloadable under an <a href="http://www.opensource.org/licenses/">OSI-approved open-source license</a> (or dedicated to the public domain) at the latest by the time of the conference.</li>
      </ol>
      <br />
     
      <p> Whatever analysis you perform need not incorporate all of the trees.  Examples of submissions that might serve as useful waypoints include:
      <ul><li>topological comparisons</li>
      <li>supertree / supernetwork / graph analysis</li>
      <li>integration of any kind of relevant metadata</li>
      <li>visualization.</ul></p>
    
      <p> The winner of the 2012 iEvoBio Challenge will be determined by the  conference participants; judging criteria to consider will include  scientific originality, use of novel techniques (including creative  integration of data), computational efficiency of your method(s), and  broad applicability of your method(s). </p>
     
      <p> Abstracts need to be submitted by June 25, 2012, to  <a href="mailto:challenge@ievobio.org">challenge@ievobio.org</a> (your submission is only received when  acknowledged by a reply email). Selected candidates will make short  oral presentations to present their projects in a &quot;Challenge  session&quot; at the conference. Cash prizes will be awarded to the  first place (USD 1500) and runner-up entries. </p>
      <p> The 2012 iEvoBio Challenge is sponsored by the <a href="http://www.opentreeoflife.org" target="_blank">Open Tree of Life</a> project, which is funded by the National Science Foundation (NSF) (#DEB-12008809).
      </p>
      <h2>Data and metadata availability requirements:</h2>
     
      <p> We strongly encourage making your dataset publicly available for  reuse. If your dataset is not already available through an online  database or data archive, we recommend depositing it in <a href="http://treebase.org/">TreeBASE</a> (or <a href="http://datadryad.org/">Dryad</a>)  with an accompanying data paper that describes it (for example, in  BMC Research Notes) if it hasn&rsquo;t already been published previously.  Alternatively, <a href="http://figshare.com/">FigShare</a> accepts data without a publication. Depositing your data in an  archive has a <a href="http://www.google.com/url?q=http%3A%2F%2Fdatadryad.org%2Fdepositing%23why&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNHqSU_005jtOEzeL6xjksClLJWnsw">variety</a> <a href="http://figshare.com/blog/Your%20research%20should%20be%20in%20a%20repository/14">of</a> <a href="http://www.plosone.org/article/info:doi%2F10.1371%2Fjournal.pone.0000308">advantages</a> over simply posting them on a website, notably that they can be  cited, and those citations can be tracked.</p>
     
      <p> If you cannot publicly release your alignments or trees by the time  of the conference, you must provide at least the following summary  metadata as part of your submission in a publicly accessible form  (e.g. a text file):
     <ul>
      <li>The number of trees in the dataset</li>
      <li>The size distribution of trees (i.e., how many taxa in each tree?)</li>
      <li>A description of the source data and how they were worked up, as  follows:</li>
      <li>The method by which characters were determined (e.g., 454  sequencing, morphological analysis)</li>
      <li>Information about any reference databases that were used</li>
      <li>Information to help understand the OTU naming</li>
      <li>Details of sequence alignment or other techniques used to prepare  the data</li>
      <li>Phylogenetic inference method, including any non-default parameter  settings</li>
      <li>Information about branch lengths and support values: were they  used? Were thresholds applied?</li></ul>
	  </p>
     
      <p> This list is guided by, and a subset of, the developing <a href="http://www.evoio.org/wiki/MIAPA">Minimum Information About a Phylogenetic Analysis</a> (MIAPA) reporting standard. Please see the <a href="http://wiki.tdwg.org/twiki/bin/view/Phylogenetics/MIAPADraft#Reconciled_draft_checklist">current draft checklist</a> for further context of the requested information, and providing  additional elements from the checklist is encouraged.</p>
    
      <p> <strong>Some eligible public datasets</strong>:</p>
      <p> This list of datasets is <strong>not exhaustive and we need your help  making it more complete!</strong> If you know of any other publicly  available datasets that contain at least 10,000 leaves in total and  are freely distributable, <strong>please email us at <a href="mailto:committee@ievobio.org">committee@ievobio.org</a>  and we&rsquo;ll post a description of the dataset and link to it here. </strong> <ul>
      
      <li>The 159,905 prokaryotic phylogenetic trees from &ldquo;Telling the  Whole Story in a 10,000-Genome World&rdquo; (<a href="http://www.google.com/url?q=http%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpubmed%2F21714939&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNF7TX7uhhfi427Y1h2fyWLtdGKA3A">PMID:21714939</a>).   Trees available at Dryad (doi:<a href="http://dx.doi.org/10.5061/dryad.gf39c8dc">10.5061/dryad.gf39c8dc</a>)</li>
      <li>10ktrees <a href="http://10ktrees.fas.harvard.edu/">http://10ktrees.fas.harvard.edu/</a></li>
      <li>Any sufficiently large subset of TreeBASE (<a href="http://www.treebase.org">www.treebase.org</a>) or  dataset(s) in Dryad (<a href="http://datadryad.org/">http://datadryad.org/</a>)</li>
      <li>AFToL datasets and trees are deposited in TreeBASE. These datasets are also <a href="http://wasabi.lutzonilab.net/pub/alignments/download_alignments">available on the AFToL website</a>.</li>
     <li><a href="http://www.graemetlloyd.com/matr.html">Data matrices and trees for dinosaurs</a> collected by Graeme T. LLoyd</li>
     <li>The <a href="http://phylota.net/pb/Download.htm">PhyLoTA project's collection of 22,000+ trees</a>. A set of 22,165 trees from nonredundant maximal clusters are available in one file. Trees in this set are built from disjoint sets of sequence data, and so are nonredundant with respect to the underlying data. Each tree is a RAxML optimal tree from a Muscle alignment (branch lengths included).</li>
</ul></p>      
      <p> <strong><em>A note about related challenge competitions:</em></strong><br/> Alongside this year&rsquo;s iEvoBio Challenge, 2012 iEvoBio sponsor <a href="http://www.biomatters.com" target="_blank">Biomatters</a> is running the <strong><a href="geneious_challenge.html">Geneious Challenge</a></strong>. Opening and closing dates are the same as for iEvoBio. Everyone is free to enter (and win!) both challenges, but do keep in  mind that all entries for the Geneious Challenge must be submitted  according to their instructions.<br/>
	In addition, the <a href="http://eol.org/" target="_blank">Encyclopedia of Life</a> (EOL), together with the <a href="http://globalnames.org/" target="_blank">Global Names</a> project, was running the <strong><a href="http://eol.org/info/tree_challenge" target="_blank">EOL Phylogenetic Tree challenge</a></strong> for producing a very large set of scientific names organized such that it can be used by EOL to allow browsing of its content by phylogenetic relationships. The two winners will receive a paid trip to the iEvoBio conference, and deadline was April 15, 2012.</p>
      <h2>2012 Challenge entrants</h2>
	<p><b>iEvoBio Challenge:</b>
	<ul><li>Brian O'Meara, Tracy Heath, Jonathan Eastman, Peter Midford, 
Joseph Brown, Matt Pennell, Mike Alfaro, Luke Harmon: <a href="https://groups.google.com/group/ievobio2012-challenge/browse_frm/thread/c303dee5666ff366?hl=en">DateLife.org</a></li>
	<li>Christopher Widden: <a href="https://groups.google.com/group/ievobio2012-challenge/browse_frm/thread/6cc3cb3918854920?hl=en">Observations from a 244-taxa Bacterial Supertree constructed to minimize lateral genetic transfer</a>
	</ul></p>
      <p>&nbsp;</p>
      <a href="#top" class="credit">top</a> </div>
    <!-- end of main div  -->
    <div id="sidemenu"> 
	<a href="about.html">About the Conference</a> 
	<a href="program.html">Program </a>  
	<span class="indent"><a href="program.html#schedule">Program Schedule</a></span> 
	<span class="indent"><a href="program.html#dates">Important Dates </a></span> 
	<a href="challenge.html">iEvoBio Challenge</a> 
	<a href="geneious_challenge.html">Geneious challenge</a>
	<a href="travel_awards.html">Travel Awards</a>
	<a href="register.html">Register</a> 
	<a href="sponsor.html">Sponsor Information</a> 
	<a href="faq.html">F.A.Q.</a> </div>
  </div>
  <!--end of content div -->
<div id="extras">

<div id="twitter">
<script>
new TWTR.Widget({
  version: 2,
  type: 'search',
  search: 'ievobio',
  interval: 6000,
  title: 'iEvoBio',
  subject: '',
  width: 250,
  height: 300,
  theme: {
    shell: {
      background: '#333333',
      color: '#f5ec3f'
    },
    tweets: {
      background: '#ffffff',
      color: '#444444',
      links: '#1985b5'
    }
  },
  features: {
    scrollbar: false,
    loop: true,
    live: true,
    hashtags: true,
    timestamp: true,
    avatars: true,
    toptweets: true,
    behavior: 'default'
  }
}).render().start();
</script>
</div>

  <div id="comments">
<div id="disqus_thread"></div>
<script type="text/javascript">
   /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
   var disqus_shortname = 'ievobio2012';

   // The following are highly recommended additional parameters. Remove the slashes in front to use.
   var disqus_identifier = 'challenge_comments';
   var disqus_url = 'http://ievobio.org/';

   /* * * DON'T EDIT BELOW THIS LINE * * */
   (function() {
       var dsq = document.createElement('script'); 
       dsq.type = 'text/javascript'; 
       dsq.async = true;
       dsq.src = 'http://' + disqus_shortname + '.disqus.com/embed.js';
       (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
   })();
</script>
<noscript>Please enable JavaScript to view the <a
href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
<a href="http://disqus.com" class="dsq-brlink">Blog comments powered
by <span class="logo-disqus">Disqus</span></a>
</div>
   
  </div>
	
  <!--end of extras div -->

<br class="clearfloat" />

  
  <div id="footer">
    <p class="credit2"><a href="http://www.twitter.com/ievobio"><img src="http://twitter-badges.s3.amazonaws.com/t_small-b.png" alt="Follow iEvoBio on Twitter" border="0"/></a> Last updated Mar. 11, 2012. Website design - <a href="http://home.comcast.net/~rpglam/portfolio/">LAM</a></p>
  </div>
  <!--end of footer div -->
</div>
<!--end of container div -->

<script type="text/javascript">
    /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
    var disqus_shortname = 'ievobio2012'; 

    /* * * DON'T EDIT BELOW THIS LINE * * */
    (function () {
        var s = document.createElement('script'); s.async = true;
        s.type = 'text/javascript';
        s.src = 'http://' + disqus_shortname + '.disqus.com/count.js';
        (document.getElementsByTagName('HEAD')[0] || document.getElementsByTagName('BODY')[0]).appendChild(s);
    }());
</script>

</body>
</html>
